charts/gardener-extension-os-coreos/templates/poddisruptionbudget.yaml

```diff
@@ -12,3 +12,6 @@ spec:
     matchLabels:
       app.kubernetes.io/name: gardener-extension-os-coreos
       app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if semverCompare ">= 1.26-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
Should we use `GitVersion` for this? It is deprecated, see https://github.com/helm/helm/blob/6d96283c0ba0f1f3aa2c80ebe7f86e98d571d5a1/pkg/chartutil/capabilities.go#L82-L83

Reply:
Ah, thanks!
Adapted.

Reply:
We need to vendor a gardener version with https://github.com/gardener/gardener/pull/8877 for this to work, The current helm version used doesn't know of this new field.
It fails with an error similar to
```
      render error in "istio-istiod/templates/poddisruptionbudget.yaml": template: istio-istiod/templates/poddisruptionbudget.yaml:13:46: executing "istio-istiod/templates/poddisruptionbudget.yaml" at <.Capabilities.KubeVersion.Version>: can't evaluate field Version in type *version.Info
      {
          s: "render error in \"istio-istiod/templates/poddisruptionbudget.yaml\": template: istio-istiod/templates/poddisruptionbudget.yaml:13:46: executing \"istio-istiod/templates/poddisruptionbudget.yaml\" at <.Capabilities.KubeVersion.Version>: can't evaluate field Version in type *version.Info",
      }
```

Reply:
> We need to vendor a gardener version with [gardener/gardener#8877](https://github.com/gardener/gardener/pull/8877) for this to work, The current helm version used doesn't know of this new field. It fails with an error similar to
> 
> ```
>       render error in "istio-istiod/templates/poddisruptionbudget.yaml": template: istio-istiod/templates/poddisruptionbudget.yaml:13:46: executing "istio-istiod/templates/poddisruptionbudget.yaml" at <.Capabilities.KubeVersion.Version>: can't evaluate field Version in type *version.Info
>       {
>           s: "render error in \"istio-istiod/templates/poddisruptionbudget.yaml\": template: istio-istiod/templates/poddisruptionbudget.yaml:13:46: executing \"istio-istiod/templates/poddisruptionbudget.yaml\" at <.Capabilities.KubeVersion.Version>: can't evaluate field Version in type *version.Info",
>       }
> ```

Ah, good to know. Then it is
/lgtm